pkg/codegen/docs/gen.go

```diff
@@ -1771,7 +1771,16 @@ func (mod *modContext) gen(fs codegen.Fs) error {
 			return err
 		}
 
-		addFile(strings.ToLower(title), buffer.String())
+		resourceFileName := strings.ToLower(title)
+		// Handle file generation for resources named `index`. We prepend a double underscore
+		// here, since this ends up resulting in route of .../<module>/index which has trouble
+		// resolving and returns a 404 in the browser, likely due to `index` being some sort
+		// of reserved keyword.
+		if resourceFileName == "index" {
+			resourceFileName = "__index"
```

Comment:
one question? could we possibly use hyphens instead of underscores? underscores are no longer supposed to be used in URLs. no worries if its not possible though.

Reply:
oh thanks. never knew that. ok i'll try that out and see if it works.

Reply:
ok using the hyphens does work :tada: